cheatsheets/Secure_Product_Design_Cheat_sheet.md

```diff
@@ -0,0 +1,92 @@
+# Secure Product Design Cheat Sheet
+
+## Introduction
+
+The purpose of Secure Product Design is to ensure that all products meet or exceed the security requirements laid down by the organisation as part of the development lifecycle and to ensure that all security decisions made about the product being developed are explicit choices and result in the correct level of security for the product being developed.
+
+## Methodology
+
+As a basic start, establish secure defaults, minimise the attack surface area, and fail securely to those well-defined and understood, defaults.
+
+Secure Product Design comes about through two processes:
+
+1. **_Product Inception_**; and
+2. **_Product Design_**
+
+The first process happens when a product is conceived, or when an existing product is being re-invented. The latter is continuous, evolutionary, and done in the agile way, close to where the code is being written.
+
+## Security Principals
+
+### 1. The principle of Least Privilege and Separation of Duties
+
+Least Privilege is a security principle that states that users should only be given the minimum amount of access necessary to perform their job. This means that users should only be given access to the resources they need to do their job, and no more. This helps to reduce the risk of unauthorized access to sensitive data or systems, as users are only able to access the resources they need. Least Privilege is an important security principle that should be followed in order to ensure the security of an organization's data and systems.
+
+Separation of duties is a fundamental principle of internal control in business and organizations. It is a system of checks and balances that ensures that no single individual has control over all aspects of a transaction. This is done by assigning different tasks to different people, so that no one person has control over the entire process. This helps to reduce the risk of fraud and errors, as well as ensuring that all tasks are completed in a timely manner. Separation of duties is an important part of any organization's internal control system, and is essential for maintaining the integrity of the organization's financial records.
+
+### 2. The principle of Defence in Depth
+
+The principle of Defence in Depth is a security strategy that involves multiple layers of security controls to protect an organization’s assets. It is based on the idea that if one layer of security fails, the other layers will still be able to protect the asset. The layers of security can include physical security, network security, application security, and data security. The goal of Defence in Depth is to create a secure environment that is resilient to attack and can quickly detect and respond to any security incidents. By implementing multiple layers of security, organizations can reduce the risk of a successful attack and minimize the damage caused by any successful attack.
```

Comment:
Same with "Defence" vs "Defense"

Reply:
However you wish to spell it, it usually is hyphenated because it is a single noun.

Reply:
100%